packages/go_router_builder/example/lib/shell_route_with_keys_example.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: public_member_api_docs
+// ignore_for_file: public_member_api_docs, unreachable_from_main
```

Comment:
should this be in the example folder analyze_options?

Reply:
The examples folder currently doesn't have its own analyze_options and introducing one is a bit of a process in the repro. I would like to defer on that for a future PR.